### DISCUSSION TITLE
ZZZ给我的开门红

### DISCUSSION THREAD

**Discussion by SSTV-10068** [share121/inter-knot#1706] [其他]
虽然我1抽出丽娜，15抽出艾莲。但是我不是很想玩绝区零是怎么回事
![屏幕截图 2024-08-12 170309](https://github.com/user-attachments/assets/a54d9950-6b12-4289-a037-dec27840d699)
![屏幕截图 2024-08-12 170300](https://github.com/user-attachments/assets/850a882e-5d44-4883-b788-0f1d822fe7ad)

**Comment by decbdV** [comment]
不想玩就退了吧，没必要强迫自己玩，不搞拉踩就行，迷茫的绳匠，祝你在未来的日子里一帆风顺。
